test/integration/dns_test.go

```diff
@@ -1,331 +1,331 @@
 package integration
 
-import (
-	"fmt"
-	"hash/fnv"
-	"net"
-	"strconv"
-	"testing"
-	"time"
+// import (
```

Comment:
delete this entire file in one commit.  Write a high severity bugzilla referencing it for @ironcladlou .

The apiservers no longer include dns.